ports/nrf/mpconfigport.h

```diff
@@ -47,7 +47,7 @@
 #define MICROPY_HELPER_REPL         (1)
 #define MICROPY_REPL_EMACS_KEYS     (0)
 #define MICROPY_REPL_AUTO_INDENT    (1)
-#define MICROPY_KBD_EXCEPTION       (0)
+#define MICROPY_KBD_EXCEPTION       (1)
```

Comment:
I think we should enable this by default for all boards.

Reply:
Yes, that makes sense.